tools/ci/stage.sh

```diff
@@ -26,6 +27,13 @@ STAGE_VITESS="vitess"
 STAGE_E2E="e2e"
 STAGE_MISC="misc"
 
+MODULES_CORE="\
+flink-cdc-cli,\
+flink-cdc-common,\
+flink-cdc-composer,\
+flink-cdc-runtime,\
+flink-cdc-connect/flink-cdc-pipeline-connectors"
```

Comment:
Could we split `flink-cdc-connect/flink-cdc-pipeline-connectors` to module `PIPELINE_CONNECTORS `?

Reply:
OK